src/intTest/resources/patient/acceptance-unknown-recipient.fhir.json

```diff
@@ -0,0 +1,97 @@
+{
+  "parameter": [
+    {
+      "name": "gpTradingPartnerCode",
+      "valueString": "AA15"
+    },
+    {
+      "name": "acceptanceCode",
+      "valueString": "A"
+    },
+    {
+      "name": "acceptanceType",
+      "valueString": "1"
+    },
+    {
+      "name": "acceptanceDate",
+      "valueString": "1992-01-13"
+    },
+    {
+      "name": "freeText",
+      "valueString": "GP NOTES GO HERE"
+    },
+    {
+      "name": "patient",
+      "resource": {
+        "managingOrganization": {
+          "identifier": [
+            {
+              "system": "https://digital.nhs.uk/services/nhais/guide-to-nhais-gp-links-documentation",
+              "value": "TES5"
```

Comment:
Oh I think I get it now. Is TES5 the "invalid" code that triggers the error? This is confusing because it appears correct. Can you make it something that is obviously wrong?

Reply:
TES5 is valid

Reply:
cypherToMailbox: ${NHAIS_MESH_CYPHER_TO_MAILBOX:XX11=fake_mailboxId
      TES5=fake_mailboxId

Reply:
the code takes the other values, the gpTradingPartnerCode I think, I used translationItems.getRecipient(), but the recipient seems to be not what I've expected - I need to check it

Reply:
I can see it, recipient is TES51, there is 1 added to it